src/clientdlg.cpp

```diff
@@ -154,8 +154,8 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     ledDelay->setAccessibleName ( tr ( "Delay status LED indicator" ) );
 
     // buffers LED
-    QString strLEDBuffers = "<b>" + tr ( "Buffers Status LED" ) + ":</b> " +
-                            tr ( "The buffers status LED shows the current audio/streaming "
+    QString strLEDBuffers = "<b>" + tr ( "Jitter Buffers Status LED" ) + ":</b> " +
```

Comment:
Question: this LED was labelled "Local Jitter Buffer" in Settings.  Either it applies only to the local buffer (i.e. a locally detected over/underrun) or it doesn't.  This wording (including the original), is vaguer - implying it could be either the server (which doesn't tell us it's had a problem as far as I know) or local buffer.

As it's being reworded, it might be wise to make it clear now.  (I notice the Settings page didn't actually _have_ WhatsThis text, just the label.)

Reply:
I don't see anything in the protocol so it must be local only.  I will reword that.  
I was a little thrown by why the "Buffers" is plural so thought it must mean both.
Will reword.

Reply:
> As it's being reworded, it might be wise to make it clear now. (I notice the Settings page didn't actually have WhatsThis text, just the label.)

In general, I agree, but I'm questioning the "As it's being reworded" for the RC phase.

Reply:
> I'm questioning the "As it's being reworded" for the RC phase.

Unfortunate as it is, if it's needed to unblock the MacOS issue, then we're stuck with the change.